test/generate.test.ts

```diff
@@ -0,0 +1,19 @@
+import { readFileSync } from 'fs'
+import { resolve } from 'path'
+import { setupTest, NuxtTestContext } from '../src'
+
+describe('generate', () => {
+  const ctx: NuxtTestContext = setupTest({
```

Comment:
We may also destructure `{ nuxt }` for tests. Will also remove requirement of importing `NuxtTestContext` type

Reply:
For some reason, when it desestruturar the test fails

Reply:
What's the bug? 😬